rasa/core/channels/channel.py

```diff
@@ -314,7 +311,7 @@ def _message(
         }
 
         # filter out any values that are `None`
-        return utils.remove_none_values(obj)
+        return {k: v for k, v in obj.items() if v is not None}
```

Comment:
I felt it doesn't really add value as the code is as understandable and I'd like to avoid putting to many useless util functions in our `shared` package as we should aim to keep that as concise as possible.